### ISSUE TITLE
Newest ULX bugging out TTT

### ISSUE THREAD

**Issue by originalwhitestar** [Nayruden/Ulysses#522]
Hello there, uhm with the newest version of ULX & ULib, TTT Seems to bug out(fresh install of server), older versions(and none) work fine, when I use the newest version it gives this error:
```
[ERROR] gamemodes/terrortown/gamemode/cl_voice.lua:666: attempt to compare number with nil
  1. Draw - gamemodes/terrortown/gamemode/cl_voice.lua:666
   2. unknown - gamemodes/terrortown/gamemode/cl_hud.lua:350
```

**Comment by Nayruden** [comment]
Is that the full stack trace and the only error? There's nothing about ULX/ULib in there.

**Comment by originalwhitestar** [comment]
I only get that error when having the newest version installed, when I use an older one, or none, my server works fine, Plus the newest version marks me as "Unauthed" which probably causes the error.

**Comment by SocDemScum** [comment]
Here's the full output from my server:

```
steamcmd@devival:~/Steam/steamapps/common/GarrysModDS$ ./srcds_run -game garrysmod +map cs_office -maxplayers 32 +gamemode terrortown -authkey 8B7B5CB99A33445848EEBCC1D2BC3FB3 +host_workshop_collection 416597860 -port 27015
Auto detecting CPU
Using default binary: ./srcds_linux
Server will auto-restart if there is a crash.

Console initialized.
Adding Path: [/home/steamcmd/Steam/steamapps/common/Counter-Strike Source Dedicated Server/cstrike]
FIND [/home/steamcmd/Steam/steamapps/common/Counter-Strike Source Dedicated Server/cstrike/*.vpk]
ADDING [/home/steamcmd/Steam/steamapps/common/Counter-Strike Source Dedicated Server/cstrike/cstrike_pak_000.vpk]
Couldn't load shader dll: game_shader_generic_garrysmod_srv.soConVarRef mat_dxlevel doesn't point to an existing ConVar
Game_srv.so loaded for "Garry's Mod"
Setting breakpad minidump AppID = 4000
Initializing Steam libraries for secure Internet server
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
dlopen failed trying to load:
/home/steamcmd/.steam/sdk32/libsteam.so
with error:
/home/steamcmd/.steam/sdk32/libsteam.so: cannot open shared object file: No such file or directory
Logging into anonymous gameserver account.
Fetching Workshop Addons..
Calling GetCollectionDetails
Connection to Steam servers successful.
   Public IP is 149.202.188.15.
Assigned anonymous gameserver Steam ID [A-1:2006144008(5932)].
VAC secure mode is activated.
Collection has 11 Addons
Mounting Addon 'Murder' (187073946)
Mounting Addon 'Melonbomber' (237537750)
Mounting Addon 'Prophunters' (260275546)
Mounting Addon 'ttt_minecraft_haven' (389346280)
Mounting Addon 'ttt_mc_skyislands' (106498411)
Mounting Addon 'Office (ph_office)' (188818807)
Mounting Addon 'PROP HUNT: Wolvin's Little Restaurant (ph_restaurant)' (186139871)
Mounting Addon 'ttt_bb_teenroom_b2' (141103402)
Mounting Addon 'ttt_freddy_the_ship' (119889220)
Mounting Addon 'ttt_waterworld' (157420728)
Mounting Addon 'md_clue' (217808514)
Adding Filesystem Addon '/home/steamcmd/Steam/steamapps/common/GarrysModDS/garrysmod/addons/ulib'
Adding Filesystem Addon '/home/steamcmd/Steam/steamapps/common/GarrysModDS/garrysmod/addons/ulx'
dlopen failed trying to load:
/home/steamcmd/.steam/sdk32/libsteam.so
with error:
/home/steamcmd/.steam/sdk32/libsteam.so: cannot open shared object file: No such file or directory
dlopen failed trying to load:
/home/steamcmd/.steam/sdk32/libsteam.so
with error:
/home/steamcmd/.steam/sdk32/libsteam.so: cannot open shared object file: No such file or directory
dlopen failed trying to load:
/home/steamcmd/.steam/sdk32/libsteam.so
with error:
/home/steamcmd/.steam/sdk32/libsteam.so: cannot open shared object file: No such file or directory
Unknown command "ttt_dyingshot"
Unknown command "ttt_karma"
Unknown command "ttt_playercolor_mode"
Unknown command "ttt_ragdoll_collide"
Unknown command "cl_cmdrate"
Unknown command "cl_updaterate"
Unknown command "rate"
Changing gamemode to Trouble in Terrorist Town (terrortown)
Network: IP 127.0.1.1, mode MP, dedicated Yes, ports 27015 SV / 27005 CL
PREP OK
///////////////////////////////
//      Ulysses Library      //
///////////////////////////////
// Loading...                //
//  shared/defines.lua       //
//  shared/misc.lua          //
//  shared/util.lua          //
//  shared/hook.lua          //
//  shared/table.lua         //
//  shared/player.lua        //
//  server/player.lua        //
//  shared/messages.lua      //
//  shared/commands.lua      //
//  server/concommand.lua    //
//  server/util.lua          //
//  shared/sh_ucl.lua        //
//  server/ucl.lua           //

[ERROR] addons/ulib/lua/ulib/shared/misc.lua:30: attempt to index local 'str' (a nil value)
  1. explode - addons/ulib/lua/ulib/shared/misc.lua:30
   2. removeCommentHeader - addons/ulib/lua/ulib/shared/misc.lua:631
    3. reloadGroups - addons/ulib/lua/ulib/server/ucl.lua:40
     4. unknown - addons/ulib/lua/ulib/server/ucl.lua:123
      5. include - [C]:-1
       6. unknown - addons/ulib/lua/ulib/init.lua:39
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/misc.lua:30: attempt to index local 'str' (a nil value)
  1. explode - addons/ulib/lua/ulib/shared/misc.lua:30
   2. removeCommentHeader - addons/ulib/lua/ulib/shared/misc.lua:631
    3. reloadGroups - addons/ulib/lua/ulib/server/ucl.lua:40
     4. unknown - addons/ulib/lua/ulib/server/ucl.lua:123
      5. include - [C]:-1
       6. unknown - addons/ulib/lua/ulib/init.lua:39
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  server/phys.lua          //
//  server/player_ext.lua    //
//  server/entity_ext.lua    //
//  shared/cami_global.lua   //
//  server/cami.lua    //
// Load Complete!            //
///////////////////////////////
[ULIB] Loading SHARED module: ulx_init.lua
///////////////////////////////
//       ULX Admin Mod       //
///////////////////////////////
// Loading...                //
//  sh_defines.lua           //
//  lib.lua                  //
//  base.lua                 //
//  sh_base.lua              //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/sh_base.lua:129
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:22
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/sh_base.lua:129
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:22
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  log.lua                  //

[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/log.lua:1
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:24
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/log.lua:1
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:24
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: slots.lua        //

[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/modules/slots.lua:1
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:28
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/modules/slots.lua:1
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:28
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: uteam.lua        //
//  MODULE: votemap.lua      //

[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/modules/votemap.lua:34
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:28
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/modules/votemap.lua:34
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:28
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: xgui_server.lua  //
///////////////////////////////
// ULX GUI -- by Stickly Man //
///////////////////////////////
// Adding Main Modules..     //
//  bans.lua                 //
//  commands.lua             //
//  groups.lua               //
//  maps.lua                 //
//  settings.lua             //
// Adding Setting Modules..  //
//  client.lua               //
//  server.lua               //
// Adding Gamemode Modules.. //
//  sandbox.lua              //
// Loading Server Modules..  //
//  sv_bans.lua              //
//  sv_groups.lua            //
//  sv_maps.lua              //
//  sv_sandbox.lua           //
//  sv_settings.lua          //
// XGUI modules added!       //
///////////////////////////////
//  MODULE: chat.lua         //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/chat.lua:11
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/chat.lua:11
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: fun.lua          //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/fun.lua:23
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/fun.lua:23
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: menus.lua        //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/menus.lua:45
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/menus.lua:45
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: rcon.lua         //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/rcon.lua:11
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/rcon.lua:11
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: teleport.lua     //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/teleport.lua:95
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/teleport.lua:95
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: user.lua         //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/user.lua:48
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/user.lua:48
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: userhelp.lua     //
//  MODULE: util.lua         //

[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/util.lua:33
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulib/lua/ulib/shared/commands.lua:1220: attempt to call field 'registerAccess' (a nil value)
  1. defaultAccess - addons/ulib/lua/ulib/shared/commands.lua:1220
   2. unknown - addons/ulx/lua/ulx/modules/sh/util.lua:33
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  MODULE: vote.lua         //

[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/modules/sh/vote.lua:6
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3


[ERROR] addons/ulx/lua/ulx/base.lua:24: attempt to call field 'registerAccess' (a nil value)
  1. convar - addons/ulx/lua/ulx/base.lua:24
   2. unknown - addons/ulx/lua/ulx/modules/sh/vote.lua:6
    3. include - [C]:-1
     4. unknown - addons/ulx/lua/ulx/init.lua:33
      5. include - [C]:-1
       6. unknown - addons/ulx/lua/ulib/modules/ulx_init.lua:2
        7. include - [C]:-1
         8. unknown - addons/ulib/lua/ulib/init.lua:72
          9. include - [C]:-1
           10. unknown - addons/ulib/lua/autorun/ulib_init.lua:3

//  end.lua                  //
// Load Complete!            //
///////////////////////////////
Included TTT language file: chef.lua
Included TTT language file: english.lua
Included TTT language file: german.lua
Included TTT language file: portuguese.lua
Included TTT language file: russian.lua
Included TTT language file: spanish.lua
Included TTT language file: swedish.lua
Included TTT language file: tradchinese.lua

[ERROR] addons/ulx/lua/ulx/xgui/server/sv_bans.lua:6: attempt to call field 'registerAccess' (a nil value)
  1. init - addons/ulx/lua/ulx/xgui/server/sv_bans.lua:6
   2. fn - addons/ulx/lua/ulx/modules/xgui_server.lua:281
    3. unknown - addons/ulib/lua/ulib/shared/hook.lua:110


[ERROR] addons/ulx/lua/ulx/xgui/server/sv_bans.lua:6: attempt to call field 'registerAccess' (a nil value)
  1. init - addons/ulx/lua/ulx/xgui/server/sv_bans.lua:6
   2. fn - addons/ulx/lua/ulx/modules/xgui_server.lua:281
    3. unknown - addons/ulib/lua/ulib/shared/hook.lua:110

ConVarRef room_type doesn't point to an existing ConVar
Executing dedicated server config file server.cfg
Using map cycle file 'cfg/mapcycle_default.txt'.  ('cfg/mapcycle.txt' was not found.)
Attempted to create unknown entity type info_ladder!
Can't init info_ladder
Attempted to create unknown entity type info_ladder!
Can't init info_ladder
Attempted to create unknown entity type func_buyzone!
Can't init func_buyzone
Attempted to create unknown entity type func_buyzone!
Can't init func_buyzone
Attempted to create unknown entity type func_hostage_rescue!
Can't init func_hostage_rescue
Attempted to create unknown entity type func_hostage_rescue!
Can't init func_hostage_rescue
Initializing Steam libraries for secure Internet server
[S_API FAIL] SteamAPI_Init() failed; SteamAPI_IsSteamRunning() failed.
dlopen failed trying to load:
/home/steamcmd/.steam/sdk32/libsteam.so
with error:
/home/steamcmd/.steam/sdk32/libsteam.so: cannot open shared object file: No such file or directory
Logging into anonymous gameserver account.
Writing cfg/banned_user.cfg.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Invalid command entered. If you need help, please type "ulx help" in your console.
Connection to Steam servers successful.
   Public IP is 149.202.188.15.
Assigned anonymous gameserver Steam ID [A-1:2006406148(5932)].
VAC secure mode is activated.
```

**Comment by SocDemScum** [comment]
The fix for #523 also fixed this glitch

**Comment by Nayruden** [comment]
Most likely fixed by 8e63db96aa36f24ac42f3180fd39d851ff0769b3. Please let me know if this is not the case.
